corehq/apps/cloudcare/static/cloudcare/js/formplayer/menus/collections.js

```diff
@@ -30,12 +30,17 @@ FormplayerFrontend.module("Menus.Collections", function (Collections, Formplayer
             'maxWidth',
             'maxHeight',
             'widthHints',
+            'useUniformUnits',
```

Comment:
yes - this wasn't being passed into the view properly before and hence case tiles were sizing incorrectly